modules/timeutil/since.go

```diff
@@ -115,13 +115,13 @@ func timeSincePro(then, now time.Time, lang translation.Locale) string {
 }
 
 // TimeSince renders relative time HTML given a time.Time
-func TimeSince(then time.Time, lang translation.Locale) template.HTML {
+func TimeSince(then time.Time, lang translation.Locale, relativeTimeOptions ...string) template.HTML {
 	timestamp := then.UTC().Format(time.RFC3339)
 	// declare data-tooltip-content attribute to switch from "title" tooltip to "tippy" tooltip
-	return template.HTML(fmt.Sprintf(`<relative-time class="time-since" prefix="%s" datetime="%s" data-tooltip-content data-tooltip-interactive="true">%s</relative-time>`, lang.Tr("on_date"), timestamp, timestamp))
+	return template.HTML(fmt.Sprintf(`<relative-time class="time-since" %s prefix="%s" datetime="%s" data-tooltip-content data-tooltip-interactive="true">%s</relative-time>`, strings.Join(relativeTimeOptions, " "), lang.Tr("on_date"), timestamp, timestamp))
```

Comment:
We should remove this `on_date`. It's not translatable.

`TimeSince` should only be used inside other translatable strings, let other strings have proper wording

Reply:
Even in English, I do not think it is fine.

In the list, it doesn't need the `on` word:

![image](https://user-images.githubusercontent.com/2114189/232035395-f5017661-90eb-4788-9dcd-d217b34b00bc.png)

Reply:
Unrelated to this PR, #24074 is tracking this

Reply:
After this PR this `on` in your screenshot will disappear

Reply:
I think it is related .... it really doesn't look right.

In old code, there is no `on` prefix of the TimeSince output either, this `on` prefix was just added recently, so I do not think it is necessary.

Reply:
> After this PR this `on` in your screenshot will disappear

I know this PR is for removing this `on`.  But ...... where else would users still see the `on`?

Reply:
`on` is still visible in issue/pr list second row for example.

Reply:
This PR adds the ability to set `tense=past`, which will make things look like `4 years ago` by default instead of `on April 14, 2023` (the exact date would be available via tooltip)

Reply:
> `on` is still visible in issue/pr list second row for example.

Is that `on` necessary? If no, I would suggest to remove it, leave the better i18n support to the future.

Reply:
@wxiaoguang I suggest you discuss this in #24074, this PR is about `tense=past`

Reply:
> @wxiaoguang I suggest you discuss this in #24074, this PR is about `tense=past`

No problem. I started this comment because the `tense=past` is going to help the `on` related problem, so IMO they are related.

I posted my opinion in #24074, let's find a solution for it.

Reply:
I agree about setting `tense=past`, I think it's the solution

Reply:
Yes, except due dates

Reply:
> Yes, except due dates

I also agree. Does this PR need to do more changes: use `tense=past` as default?

About `due dates`: actually it's not related to relative-time, it should be a date(day) string.

Reply:
I understand, I will make another PR suggesting changing the default. I don't like to make significant changes in one PR, to make reviews easy

Reply:
I guess it only needs a few lines change:

```go
if len(relativeTimeOptions) == 0 { tense=past }
```

Does it work?

Reply:
The code is small, the implications are not

Reply:
If the date was in the future, it probably rendered `in X days`. With `tense=past` it will render `now`. That's what I'm worried about...

BTW feel free to push changes here, I won't block (you have better historical context here then me)

Reply:
Yes, the `on <date>` format is new with the element. We could for now opt to always set past tense and remove the `on` translation to restore old behaviour, but I do prefer the `on` format for far-past dates as well, but it needs to be worked out first how to correctly translate that format (https://github.com/go-gitea/gitea/issues/24074).

Reply:
> feel free to push changes here,

Thank you very much for the trust, I will try it myself to see whether I really understand the problem (see you in a few hours) 😄

Reply:
> If the date was in the future, it probably rendered `in X days`. With `tense=past` it will render `now`. That's what I'm worried about...
> 
> BTW feel free to push changes here, I won't block (you have better historical context here then me)

That's no problem, GH also did that.